.gitmodules

```diff
@@ -66,3 +63,9 @@
 [submodule "src/external/SPIRV-Cross"]
 	path = src/external/SPIRV-Cross
 	url = https://github.com/vita3k/SPIRV-Cross
+[submodule "src/external/pugixml"]
+	path = src/external/pugixml
+	url = https://github.com/zeux/pugixml
+[submodule "src/external/vita-toolchain"]
```

Comment:
don't need change this submodule place ^^

Reply:
dont mind it, i have a hard time remove/add that submodule